README.md

```diff
@@ -178,54 +228,6 @@ or
 curl localhost:5000//query_embedding_drift
```

Comment:
Double `//`?

Reply:
Paging @osw282  for more insights on this.

Reply:
Go with a single, it looks a bit bizarre